base/applications/mscutils/servman/mainwnd.c

```diff
@@ -590,11 +594,12 @@ MainWndCommand(PMAIN_WND_INFO Info,
         break;
 
         case ID_ABOUT:
-            DialogBox(hInstance,
-                      MAKEINTRESOURCE(IDD_ABOUTBOX),
-                      Info->hMainWnd,
-                      AboutDialogProc);
-            SetFocus(Info->hListView);
+            LoadStringW(hInstance, IDS_APPNAME, szAppName, sizeof(szAppName) / sizeof(WCHAR));
```

Comment:
Use either `ARRAYSIZE(szAppName)` or `_countof(szAppName)`, whichever is used elsewhere in this app.

Reply:
ok